smarkets/streaming_api/framing.py

```diff
@@ -82,9 +82,9 @@ def uleb128_decode(string):
 
     while True:
         try:
-            current_byte = indexbytes(string, position)
+            current_byte = byte_array[position]
         except IndexError:
```

Comment:
Why do we use `bytearray` for decoding? I can see that using `str` is bogus, but `bytes` (immutable `bytearray`) seems more appropriate here.